db/schema.rb

```diff
@@ -10,34 +10,49 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20_171_104_082_718) do
+ActiveRecord::Schema.define(version: 20171107235516) do
+
   # These are extensions that must be enabled in order to support this database
-  enable_extension 'plpgsql'
+  enable_extension "plpgsql"
+
+  create_table "games", force: :cascade do |t|
+    t.string   "name"
+    t.datetime "created_at",      null: false
+    t.datetime "updated_at",      null: false
+    t.integer  "game_id"
```

Comment:
We won't need a `game_id` inside of `game`. Rails auto adds an `id` column for all tables.

We do need a `game_id` inside of Pieces though. 😄

Reply:
We did not add the `game_id` in games that was already there.